plugins/caffeine/caffeine-output.c

```diff
@@ -112,6 +114,14 @@ static void *caffeine_create(obs_data_t *settings, obs_output_t *output)
 	return context;
 }
 
+static void caffeine_update_av(void *context, struct obs_video *video, struct obs_audio *audio)
```

Comment:
See other comment. Because these parameters are of an unknown and different type, static analyzers currently think that this PR is bad.